dev-lang/lfe/lfe-1.2.1.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+
+inherit multilib
+
+DESCRIPTION="Lisp-flavoured Erlang"
+HOMEPAGE="http://lfe.github.io/"
+SRC_URI="https://github.com/rvirding/lfe/archive/v${PV}.zip"
```

Comment:
This will result in `v1.2.1.zip` being created in DISTDIR which is a very bad name. Please rename it using `->`, preferably to `${P}...`. Also, please use .tar.gz rather than .zip.